app/package.json

```diff
@@ -19,6 +19,7 @@
     "js"
   ],
   "dependencies": {
+    "cross-env": "^5.1.3",
```

Comment:
since we are installing cross-env locally here, shouldn't the start and dev script be `./node_modules/cros-env/...`

Reply:
npm and Yarn automatically resolve that for you.